app/javascript/packs/sample.js

```diff
@@ -0,0 +1,29 @@
+let pokemon_sample = document.getElementById("sample");
+let pokemon = pokemon_sample.getElementsByClassName("pokemon-card");
+
+pokemon[0].style.display = 'block';
+let position = 0;
+let prev = document.getElementById("prev-pokemon");
+let next = document.getElementById("next-pokemon");
+
+next.addEventListener('click', () => {
+    pokemon[position].style.display = 'none';
+
+    if (position + 1 === pokemon.length ) {
+        position = 0
+    } else {
+        position += 1
+    }
+    pokemon[position].style.display = 'block';
+});
+
```

Comment:
This code is not DRY, but I've decided not to spend lot of time trying to clean it.

Reply:
That's okay sometimes https://overreacted.io/goodbye-clean-code/